en/lessons/ecto/querying.md

```diff
@@ -16,7 +16,7 @@ We can perform simple queries directly against our `Friends.Repo` with the help
 
 ### Fetching Records by ID
 
-We can use the `Repo.get/3` function to fetch a record from the database given its ID. This function requires two arguments: a "queryable" data structure and the ID of a record to retrieve from the database. It returns a struct describing the record found, if any. It returns `nil` if no such record is found.
+We can use the `Repo.get/2` function to fetch a record from the database given its ID. This function requires two arguments: a "queryable" data structure and the ID of a record to retrieve from the database. It returns a struct describing the record found, if any. It returns `nil` if no such record is found.
```

Comment:
The `Repo.get` function does in fact have an arity of three, see the docs here https://hexdocs.pm/ecto/Ecto.Repo.html#c:get/3. The third argument is optional and we're not using it in this example.

Reply:
yes, I know the arities of the function.
But the arity which the sample is using is 2 no?

Reply:
Ah okay I see what you're saying. So, while its true that we're calling this function with only two arguments, when we refer to the official function as it is defined, we refer to the correct definition/API of that function. Does that make sense?

Reply:
I think don't make sense because the example are showed is with the two like the others and is confusing. I think is better to accordingly with the sample. But that's ok..

Reply:
```suggestion
We can use the `Repo.get/3` function to fetch a record from the database given its ID. This function requires two arguments: a "queryable" data structure and the ID of a record to retrieve from the database. It returns a struct describing the record found, if any. It returns `nil` if no such record is found.
```